frontend/src/toolbar/stats/HeatmapStats.tsx

```diff
@@ -41,58 +91,53 @@ export function HeatmapStats(): JSX.Element {
                         Found: {countedElements.length} elements / {clickCount} clicks!
                     </div>
                     <div>
-                        <LemonButton
-                            icon={<IconSync />}
-                            type={'secondary'}
-                            status={'primary-alt'}
-                            size={'small'}
+                        <ButtonWithTooltipInStackingContext
+                            canLoadMoreElementStats={canLoadMoreElementStats}
                             onClick={() => loadMoreElementStats()}
-                            disabled={!canLoadMoreElementStats}
-                        >
-                            Load more
-                        </LemonButton>
-                        {canLoadMoreElementStats ? null : (
-                            <AlertMessage type={'info'} className={'mt-2'}>
-                                Loaded all elements in this data range.
-                            </AlertMessage>
-                        )}
+                            popupContainer={() => buttonWindowRef?.current ?? document.body}
+                        />
+                    </div>
+
+                    <Tooltip
+                        title={
+                            'Matching links by their target URL can exclude clicks from the heatmap if the URL is too unique.'
+                        }
+                        getPopupContainer={() => buttonWindowRef?.current ?? document.body}
+                    >
+                        <div>
+                            <LemonSwitch
+                                checked={matchLinksByHref}
+                                label={'Match links by their target URL'}
```

Comment:
When I read this switch label (on the screencast) "match links with urls? 🧐", I got confused. The tooltip also didn't help. Unfortunately I'm also a developer who struggles with writing clear interface text. Fortunately we have @Twixes 😅